src/math/rpy.hpp

```diff
@@ -7,6 +7,7 @@
 
 #include "pinocchio/math/fwd.hpp"
 #include "pinocchio/math/comparison-operators.hpp"
+#include "pinocchio/multibody/fwd.hpp"
 
 #include <Eigen/Geometry>
```

Comment:
This should be moved to the implementation file.

Reply:
Done